### PR TITLE
Add v1.0.1 of mattermost-plugin-apps to the Marketplace

### PR DESCRIPTION
**Pull request by mickmister**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-plugin-apps/issues/319

### CODE REVIEW

**Review by catalintomai — APPROVED** [review]

File: plugins.json [hunk 1]
```diff
@@ -2759,6 +2759,56 @@
...
+      "name": "Apps",
+      "description": "This plugin powers the Mattermost Apps Framework and is required for any Apps to run",
+      "homepage_url": "https://mattermost.com/marketplace/app-framework",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-apps/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-apps/releases/tag/v1.0.1",
+      "version": "1.0.1",
+      "min_server_version": "6.6.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
```
[the remark this answers is not in the document]
Comment (mickmister) [reply plugins.json:2784]: @catalintomai https://github.com/mattermost/mattermost-marketplace/pull/238

**Review by levb — APPROVED** [review]

**Comment by mickmister** [comment]
@hanzei We're adding this to the marketplace in order to install the plugin on cloud servers before they receive the commit with the prepackaged plugin

File: plugins.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (catalintomai) [reply plugins.json:2784]: @mickmister ,thanks, good to know.

**Review by DHaussermann — APPROVED** [review]
Tested and passed 
- This is behaving as expected. The plugin is available even on server where the binary is not prepackaged
- Tested upgrade Path 
- Tested Fresh install patch
- Sanity check the plugin can be enabled / disable and commands work
- Not directly related to this PR but, also confirmed listing in the marketplace works with existing Split IO mechanism to control when an upgrade does or does not occur. 

LGTM!

**Pull request merged**